client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -257,6 +257,24 @@ export const DetailView: FC = () => {
 
   if (!isSuccess && !isError) return <DetailViewSkeleton />;
 
+  // For Immunization Programs, we display as two different tabs - Vaccinations
+  // card, and the normal "Encounter" page (if defined)
+  const tabs = [];
+  if (encounter && encounter.programEnrolment?.isImmunisationProgram) {
+    tabs.push({
+      Component: (
+        <VaccinationsTab
+          encounterId={encounter.id}
+          programEnrolmentId={encounter.programEnrolment.id}
+          clinician={encounter.clinician ?? undefined}
+        />
+      ),
+      value: t('label.vaccinations'),
+    });
+    if (encounter.document.documentRegistry?.uiSchema.elements.length > 0)
+      tabs.push({ Component: JsonForm, value: t('label.encounter') });
```

Comment:
So this is the only way I could figure out how to tell if there are "boxes" to appear on the Encounter tab. I'm assuming that, initially, Vaccine program encounters will have no UI components defined?